portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,21 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Mick Toyingsirikul</h1>
+      <img src="./images/doodle.png" alt = "Avatar picture"/>
+      <p>Hello 👋! My name is Mick, I'm a 4th year CS student at the University of British Columbia.</p>
+      <ul>
```

Comment:
Do these do anything? Just a list of you being cool? haha

Reply:
Oh wait, I was trying to play with <ul> tag.  I did not request a PR but why does it show up here. I guess I should not push the test code.

Reply:
oh I think it's because you pushed it to a branch that already had a pull request. 

I think we should have merged the original PR that I had approved of updating the README. then create a new PR for modifying the page contents.